packages/terra-content-container/src/ContentContainer.jsx

```diff
@@ -54,7 +61,7 @@ const ContentContainer = ({
     <div {...customProps} className={contentLayoutClassNames}>
       {header && <div className={cx('header')}>{header}</div>}
       <div className={cx('main')}>
-        <Scroll className={cx('normalizer')} refCallback={scrollRefCallback}>
+        <Scroll className={cx('normalizer')} refCallback={scrollRefCallback} tabIndex={setFocusOnContainer ? '0' : '-1'}>
```

Comment:
Can we add tab-index to above div instead of adding it on scroll. so that we can add focus styles for class `main` similar to other terra-components like below 
<img width="275" alt="Screenshot 2023-05-26 at 11 31 55 AM" src="https://github.com/cerner/terra-core/assets/12869809/caee2426-9d71-4e61-9017-7df34d34dc9f">

Reply:
Added the role region and added the border style as per discussion